### PR TITLE
Revert "Issue#609 Add Deviations to Box Manifests (#500)"

### PR DESCRIPTION
**Pull request by Gbarra9**
This reverts commit 200e2ad0722b82362291ff8b57204ee0a13b3ebf.

- https://github.com/episphere/biospecimen/pull/500

### CODE REVIEW

File: src/events.js [hunk 1]
```diff
@@ -3187,11 +3170,12 @@ export const populateTrackingQuery = async (boxIdAndBagsObj) => {
     document.getElementById("forTrackingNumbers").innerHTML = toBeInnerHTML;
 }
 
-export const addEventCompleteButton = async (boxIdAndBagsObj, userName, boxWithTempMonitor) => {
+export const addEventCompleteButton = (boxIdAndBagsObj, userName, tempChecked) => {
```
Comment (Gbarra9) [inline src/events.js:3173]: Note to self this may need to be boxWithTempMonitor.

Comment (we-ai) [inline src/events.js:3173]:
```suggestion
export const addEventCompleteButton = async (boxIdAndBagsObj, userName, boxWithTempMonitor) => {
```

File: src/pages/shipping.js [hunk 2]
```diff
@@ -335,16 +333,12 @@ export const boxManifest = async (boxId, userName) => {
...
- */
-export const shippingManifest = async (boxIdArray, userName, isTempMonitorIncluded, currShippingLocationNumber) => {
+
+
+export const shippingManifest = async (boxesToShip, userName, tempMonitorThere, currShippingLocationNumber) => {    
+    //let tempMonitorThere = document.getElementById('tempMonitorChecked').checked;
     let response = await  getBoxes();
-    let boxArray = response.data;
+    let boxList = response.data;
     let boxIdAndBagsObj = {};
```
Comment (Gbarra9) [inline src/pages/shipping.js:342]: Previous Change needs to be kept

File: src/pages/shipping.js [hunk 3]
```diff
@@ -335,16 +333,12 @@ export const boxManifest = async (boxId, userName) => {
...
- * @param {boolean} isTempMonitorIncluded 
- * @param {*} currShippingLocationNumber 
- */
-export const shippingManifest = async (boxIdArray, userName, isTempMonitorIncluded, currShippingLocationNumber) => {
+
+
+export const shippingManifest = async (boxesToShip, userName, tempMonitorThere, currShippingLocationNumber) => {    
+    //let tempMonitorThere = document.getElementById('tempMonitorChecked').checked;
     let response = await  getBoxes();
-    let boxArray = response.data;
```
Comment (Gbarra9) [inline src/pages/shipping.js:347]: Keep previous change

File: src/pages/shipping.js [hunk 4]
```diff
@@ -335,16 +333,12 @@ export const boxManifest = async (boxId, userName) => {
...
- * @param {string[]} boxIdArray 
- * @param {string} userName 
- * @param {boolean} isTempMonitorIncluded 
- * @param {*} currShippingLocationNumber 
- */
-export const shippingManifest = async (boxIdArray, userName, isTempMonitorIncluded, currShippingLocationNumber) => {
+
+
+export const shippingManifest = async (boxesToShip, userName, tempMonitorThere, currShippingLocationNumber) => {    
+    //let tempMonitorThere = document.getElementById('tempMonitorChecked').checked;
```
Comment (we-ai) [inline src/pages/shipping.js:339]:
```suggestion
/**
 * 
 * @param {string[]} boxIdArray 
 * @param {string} userName 
 * @param {boolean} isTempMonitorIncluded 
 * @param {*} currShippingLocationNumber 
 */
export const shippingManifest = async (boxIdArray, userName, isTempMonitorIncluded, currShippingLocationNumber) => {
```

File: src/pages/shipping.js [hunk 5]
```diff
@@ -335,16 +333,12 @@ export const boxManifest = async (boxId, userName) => {
...
     let response = await  getBoxes();
-    let boxArray = response.data;
+    let boxList = response.data;
```
Comment (we-ai) [inline src/pages/shipping.js:341]:
```suggestion
    let boxArray = response.data;
```

**Review by we-ai — APPROVED** [review]

**Pull request merged**